File: app/src/main/java/com/prideven/android/hungryeats/SearchActivity.java
package com.prideven.android.hungryeats;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class SearchActivity extends AppCompatActivity {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.restaurant_list);

        ImageView mcd= findViewById(R.id.mcd);

        mcd.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                startActivity(new Intent(v.getContext(), RestaurantActivity.class));
            }
        });


    }
}

File: app/src/main/java/com/prideven/android/HungryEats/SearchActivity.kt
package com.prideven.android.hungryeats

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.prideven.android.hungryeats.RestaurantActivity

class SearchActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.restaurant_list)
        val mcd = findViewById<ImageView>(R.id.mcd)
        mcd.setOnClickListener { v ->
            startActivity(
                Intent(
                    v.context,
                    RestaurantActivity::class.java
                )
            )
        }
    }
}